update.rb

```diff
@@ -160,6 +160,8 @@ def perform_update
           $stderr.puts "RubyMotion pre-release update installed in /Library/RubyMotionPre\n\n"
         else
           $stderr.puts "Software update installed.\n\n"
+          $stderr.puts "Run `motion repo` to synchronize templates."
+          $stderr.puts "Run `rake clean:all` on your apps to remove stale files.\n\n"
```

Comment:
@amirrajan I added the reminders to display after a successful, normal software update. Let me know if you think this needs any more changes.